makefile

```diff
@@ -0,0 +1,59 @@
+#REQUIREMENTS BEFORE RUNNING MAKE ALL: ruby, yarn, docker, graphviz, imagemagick, nvm, postgres
+.PHONY: test
+
+all: install_dependencies docker setup_env setup_db seed_db create_user run
+
+test_all: update test
+
+docker: setup_docker_network setup_docker_container setup_env
+
+run:
+	docker start placecal-db || true
+	bin/dev
+
+down:
+	docker stop placecal-db
+
+# does not remove postgres image, just placecal container & network
+clean:
+	docker stop placecal-db || true
+	docker container rm placecal-db || true
+	docker network rm placecal-network
+
+install_dependencies:
+	yarn
+	bundle install
+
+setup_docker_network:
+	docker network create placecal-network
+
+setup_docker_container:
+	docker create --name placecal-db --network placecal-network --network-alias postgres -p 5432:5432 --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -e 'POSTGRES_DB=placecal_dev' -e 'POSTGRES_USER=postgres' -e 'PGUSER=postgres' -e 'POSTGRES_PASSWORD=foobar' -e 'POSTGRES_PORT=5432' postgres:14.1
+	docker start placecal-db 
+
+setup_env:
+	cp .env.example .env
+	echo POSTGRES_HOST=localhost >> .env
+	echo POSTGRES_USER=postgres >> .env
+	echo PGPASSWORD=foobar >> .env
+	echo PGHOST=localhost >> .env
+	echo PGUSER=postgres >> .env
+
+setup_db:
+	bundle exec rails db:setup db:migrate db:seed
+	bundle exec rails import:all_events
+
+seed_db:
```

Comment:
This is cloning production database to your local env which is not really seeding with minimal data. You also call this step after calling `setup_db` which also runs the `rails db:seed` command.

Reply:
From what I understood from speaking with everyone, this is the minimal amount of data required right now to be able to properly work on any of the normal bug tickets that come up for us, but if there's a more minimal setup we can do let me know and I can change this! 

I could name this more explicitly to be something like `seed_GFSC_prod_db`?

Reply:
Sure, renaming it to `seed_GFSC_prod_db` is fine.

When we get seeds working (i.e. loading neighbourhood data) then the seed process will be complete. Cloning the production database can introduce problems if we become too dependent on the data in production for our code to work.